discord/member.py

```diff
@@ -402,6 +402,22 @@ def permissions_in(self, channel):
         """
         return channel.permissions_for(self)
 
+    def has_role(self, role_id):
```

Comment:
Maybe allow you to pass the name of the role or the role object itself too? That would allow you to use the function in all decorators as in some you still have to use `discord.utils.get()`